setup.py

```diff
@@ -25,9 +25,9 @@
     install_requires=['nameko>=2.11,<3.0'],
     extras_require={
         'dev': [
-            'pytest==3.2.5',
-            'flake8',
-            'coverage',
+            'pytest==4.3.0',
+            'flake8>=3.7.6',
```

Comment:
Do we need to pin this and `coverage` with such precise versions?

Reply:
Not really. These are currently the latest ones but I don't think it's needed.
I'll change it back.